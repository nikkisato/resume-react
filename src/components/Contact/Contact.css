.contact {
width: 250px;
}