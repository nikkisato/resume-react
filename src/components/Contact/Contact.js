import React from 'react';
import data from '../resume2';
import styles from './Contact.css';

export default function Contact() {
  return (
    <div className={styles.contact}>
      <p>
        <b>Profile:</b> {data.Profile.summary}{' '}
      </p>
      <p>
        <b>Email: </b>
        {data.contact[0].email}
      </p>
      <p>
        <b>Website:</b>
        {data.contact[0].website}
      </p>
      <p>
        <b>Phone:</b> {data.contact[0].Phone}
      </p>
    </div>
  );
}
