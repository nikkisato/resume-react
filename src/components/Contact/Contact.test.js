import React from 'react';
import { shallow } from 'enzyme';
import Contact from './Contact';

describe('Contact component', () => {
  it('renders Contact', () => {
    const wrapper = shallow(<Contact />);
    expect(wrapper).toMatchSnapshot();
  });
});
