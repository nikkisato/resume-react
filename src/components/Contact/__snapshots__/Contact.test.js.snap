// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component renders Contact 1`] = `
<div
  className="contact"
>
  <p>
    <b>
      Profile:
    </b>
     
    Full Stack Developer with a passion for front-end web development and design. Programming is one of the ways to express my creativity and I look forward to being able to hone my skills in a collaborative and dynamic environment.
     
  </p>
  <p>
    <b>
      Email: 
    </b>
    Nikki.satopdx@gmail.com
  </p>
  <p>
    <b>
      Website:
    </b>
    www.nikkisato.com
  </p>
  <p>
    <b>
      Phone:
    </b>
     
    (503)-858-7070
  </p>
</div>
`;
